src/frontend/src/components/identityCard.ts

```diff
@@ -30,14 +30,14 @@ export class IdentityBackground {
 
   // Only the src attribute is used so the same Element can be shared across Templates
   public static singleton?: IdentityBackground;
-  public static getSingleton(): IdentityBackground {
-    IdentityBackground.singleton ??= new IdentityBackground();
+  public static getSingleton(image?: string): IdentityBackground {
```

Comment:
what happens if you call `getSingleton` twice in a row with different parameters?

Reply:
Good point, the first one will be used. Let me improve it in another PR.

Reply:
I'm not sure how a singleton would work with an argument but I'll stay tuned and learn something!